### PR TITLE
Socialite implemented

### PR DESCRIPTION
**Pull request by rathorevaibhav**
Implemented socialite so that only organization users can access data.

### CODE REVIEW

**Review by nishanth-kd — APPROVED** [review]
Why do we need socialite for the weeklydose package?

**Comment by rathorevaibhav** [comment]
I used it because default Laravel auth allows anyone to register and login. For internal use, we only need to allow our team. Since we're using GSuite, I figured it'll be better to have OAuth.

On deeper look, using socialite here can be a bit organization specific. I think we should keep the default Laravel auth in this repo and use socialite in a forked one? Thoughts?

File: resources/views/welcome.blade.php [hunk 1]
```diff
@@ -71,8 +71,7 @@
                     @auth
                         <a href="{{ url('/reading-items') }}">Home</a>
                     @else
-                        <a href="{{ route('login') }}">Login</a>
-                        <a href="{{ route('register') }}">Register</a>
```
Comment (nishanth-kd) [inline resources/views/welcome.blade.php:75]: We need to remove the register route from the listed Routes as well. Hiding the action from view would

Comment (rathorevaibhav) [reply resources/views/welcome.blade.php:75]: Created a tracking #7. Will target separately.

**Comment by rathorevaibhav** [comment]
Based on our discussion, we figured it really depends on an organization what to use. Either go with socialite or use Laravel's default auth. Some may go for former, while others can go for the latter. We can provide a config on which auth mechanism to use. Tracking this in #6

**Pull request merged**